### ISSUE TITLE
Add option to attach menu node to `<body>`

### ISSUE THREAD

**Issue by ivansiacho** [ericgio/react-bootstrap-typeahead#95]
Hi, is there anyway to create the options list as a part of `<body>` level
right now I've this Typeahead inside to `overflow:auto` div, and this list is getting clipped

![clipped-list](https://cloud.githubusercontent.com/assets/5598171/19581215/231de08c-96f1-11e6-8658-59bd22215799.png)

Regards,

**Comment by ericgio** [comment]
Try setting the `align` [prop](https://github.com/ericgio/react-bootstrap-typeahead#props) to either 'left' or 'right'.

**Comment by ivansiacho** [comment]
No i'm not referring to list items, is about list container. 
The list container is going to be created below of input, so if this typeahead is into parent who has `overflow:hidden` it's going to be clipped

**Comment by ericgio** [comment]
Oh I see. No, there's currently no way to attach the menu to body, unfortunately.

**Comment by nareshbhatia** [comment]
The way this issue is usually solved is to provide a property to attach the control to any container that the component user wants to. See [react-bootstrap-date-picker](https://github.com/pushtell/react-bootstrap-date-picker) as an example - specifically the `calendarContainer` property. The example [here](http://pushtell.github.io/react-bootstrap-date-picker/) clearly shows the issue and the solution (look under "Placed in a Scrolling Container"). The solution simply uses the `container` property of the bootstrap Overlay: https://github.com/pushtell/react-bootstrap-date-picker/blob/master/src/index.jsx#L392. @ericgio,  is there a similar solution possible?

**Comment by ericgio** [comment]
I'm sure it's possible, just a question of how much work it will be :) I'm not currently using `react-overlays`, but I imagine that would make things much easier.

**Comment by ericgio** [comment]
Thanks for the example, btw I'll take a look.

**Comment by ericgio** [comment]
This looks doable, but will take a bit of work. I'll get to it when I have a chance.

**Comment by fbrullo** [comment]
I am facing the same problem. Please let us know when it will be fixed.

**Comment by ericgio** [comment]
This is now available in `v1.0.0-beta1` by using the `bodyContainer` prop. Use at your own risk, as there are [breaking changes](https://github.com/ericgio/react-bootstrap-typeahead/releases/tag/v1.0.0-beta1) and some of the features haven't been stress tested yet.

I'll close this task out when the stable release is published.

**Comment by nareshbhatia** [comment]
Thanks @ericgio, much appreciated. Will let you know if we see any issues.

**Comment by ivansiacho** [comment]
Hi @ericgio i'm getting an error with this new version in all my typeahead components. 
`Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.` 
Is there any other dependency that we need to upgrade too?

**Comment by ericgio** [comment]
I removed the `default` export, so my guess is you need to update how you require/import:

```
// v0.x.x
import Typeahead from 'react-bootstrap-typeahead'; // ES2015
var Typeahead = require('react-bootstrap-typeahead').default; // CommonJS

// v1.0
import {Typeahead} from 'react-bootstrap-typeahead'; // ES2015
var Typeahead = require('react-bootstrap-typeahead').Typeahead; // CommonJS
```

I'll add this to the v1.0.0 section of the README.

**Comment by ericgio** [comment]
This feature is now available in v1.0.0.

**Comment by nareshbhatia** [comment]
Awesome! Thanks, @ericgio.
